addons/mail/static/src/discuss/core/common/attachment_panel.js

```diff
@@ -0,0 +1,63 @@
+/* @odoo-module */
+
+import { DateSeparator } from "@mail/core/common/date_separator";
+import { ActionPanel } from "@mail/discuss/core/common/action_panel";
+import { AttachmentList } from "@mail/core/common/attachment_list";
+
+import { Component, onWillStart, onWillUpdateProps } from "@odoo/owl";
+import { _t } from "@web/core/l10n/translation";
+import { useService } from "@web/core/utils/hooks";
+import { useVisible } from "@mail/utils/common/hooks";
+
+/**
+ * @typedef {Object} Props
+ * @property {import("@mail/model/model").Thread} thread
+ */
+export class AttachmentPanel extends Component {
+    static components = { ActionPanel, AttachmentList, DateSeparator };
+    static props = ["thread"];
+    static template = "mail.AttachmentPanel";
+
+    setup() {
+        this.threadService = useService("mail.thread");
+        this.attachmentUploadService = useService("mail.attachment_upload");
+        onWillStart(() => {
+            this.threadService.fetchMoreAttachments(this.props.thread);
+        });
+        onWillUpdateProps((nextProps) => {
+            if (nextProps.thread.localId !== this.props.thread.localId) {
+                this.threadService.fetchMoreAttachments(nextProps.thread);
+            }
+        });
+        const loadOlderState = useVisible("load-older", () => {
+            if (loadOlderState.isVisible) {
+                this.threadService.fetchMoreAttachments(this.props.thread);
+            }
+        });
+    }
+
+    get title() {
+        return _t("Attachments");
+    }
```

Comment:
I think with recent improved `_t()`, you can just `<t t-set t-value/>` in template.
This is one less indirection if this works

Reply:
AFAIK `t-value` is not exported, but you can do something like this:
```xml
<t t-set="title">Attachments</t>
```